infrastructure/project.py

```diff
@@ -0,0 +1,24 @@
+import pulumi
+from pulumi_gcp.projects import Service, IAMMember
+from pulumi_gcp.serviceaccount import Account
+
+PROJECT = pulumi.Config("gcp").require("project")
+REGION = pulumi.Config("gcp").require("region")
+
+Service("enable-run", project=PROJECT, service="run.googleapis.com")
+Service("enable-eventarc", project=PROJECT, service="eventarc.googleapis.com")
+Service("enable-secretmanager", project=PROJECT, service="secretmanager.googleapis.com")
+Service("enable-pubsub", project=PROJECT, service="pubsub.googleapis.com")
+
+svc_account = Account(
```

Comment:
Nitpick: I'm preferential to not using truncations/acronyms (e.g. `svc`) in variable names and prefer spelling everything out.